packages/lib-react-components/src/ZooFooter/components/PolicyLinkSection/PolicyLinkSection.js

```diff
@@ -0,0 +1,35 @@
+import { Anchor, Box } from 'grommet'
+import { arrayOf, string} from 'prop-types'
+import React from 'react'
+import { zipWith } from 'lodash'
+import zooTheme from '@zooniverse/grommet-theme'
+
+import zipLabelsAndUrls from '../../helpers/zipLabelsAndUrls'
+import SpacedText from '../../../SpacedText'
+
+export default function PolicyLinkSection ({ labels, urls }) {
+  const links = zipLabelsAndUrls(labels, urls)
+
+  return (
+    <Box direction='row' gap='medium' tag='nav'>
+
+      {links.length > 0 && links.map(link => (
+        <Anchor
+          href={link.url}
+          key={link.url}
+          size='small'
```

Comment:
The `SpacedText` component is setting its own font size and is ignoring this. I think `SpacedText` needs to be set `size='xsmall'`